core/kt-osrd-sim-infra/src/main/kotlin/fr/sncf/osrd/sim_infra/api/PathProperties.kt

```diff
@@ -62,18 +62,23 @@ interface PathProperties {
 
 /** Build a Path from chunks and offsets, filtering the chunks outside the offsets */
 fun buildPathPropertiesFrom(
-    infra: TrackProperties,
+    infra: RawSignalingInfra,
     chunks: DirStaticIdxList<TrackChunk>,
     pathBeginOffset: Offset<Path>,
     pathEndOffset: Offset<Path>,
+    routes: List<String>? = null
```

Comment:
I kept the route IDs as String in `SpeedSection` because there doesn't seem to be a way to get the right route index in the InfraAdapter, and since I believe PE is rewriting that part, I'm not really sure about adding a whole reference resolving step. Is it ok?